packages/firestore/src/core/firestore_client.ts

```diff
@@ -536,7 +536,10 @@ export class FirestoreClient {
   }
 
   unlisten(listener: QueryListener): void {
-    this.verifyNotShutdown();
+    // Make unlisten a no-op if client is shut down.
+    if(this.clientShutdown) {
```

Comment:
There is something going on with your formatter. This should be `if (this.clientShutdown)`

I would also slightly prefer if you wrote the inverse and removed the early return. Optional.

Reply:
Yeah, it's strange, the automated prettier commit did not make into the push somehow..